projects/hipblaslt/tensilelite/Tensile/Components/CustomSchedule.py

```diff
@@ -72,6 +73,131 @@ def get_code(self):
         return [item[1] for item in self.schedule]
 
 
+def verifyLRsDoneInTime(schedule_info: 'ScheduleInfo', context: dict) -> tuple[bool, str]:
```

Comment:
```suggestion
def verifyLRsCompleteBeforeRegisterRead(schedule_info: 'ScheduleInfo', context: dict) -> tuple[bool, str]:
```

or something, just because the LRs also need to be 'done in time' for the GRs to start.